.stylelintrc

```diff
@@ -1,6 +1,6 @@
 extends: stylelint-config-standard
 
 rules:
-  color-hex-case: upper
+  color-hex-case: lower
```

Comment:
The npm run format converted all the hex to lower, so i flipped this rule to make everyone's life easier